src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1589,7 +1583,7 @@ namespace HeatBalFiniteDiffManager {
             state.dataHeatBalSurf->SurfOpaqOutFaceCondFlux(Surf) = -QNetSurfFromOutside;
             state.dataHeatBalFiniteDiffMgr->QHeatOutFlux(Surf) = QNetSurfFromOutside;
 
-        } else if (surface_ExtBoundCond <= 0) { // regular outside conditions
+        } else { // regular outside conditions
```

Comment:
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:1592]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 1525.